op-monitorism/transaction_monitor/monitor.go

```
@@ -0,0 +1,377 @@
+	Type   CheckType              `yaml:"type"`
+	Params map[string]interface{} `yaml:"params"`
+}
+
+type WatchConfig struct {
+	Address common.Address `yaml:"address"`
+	Filters []CheckConfig  `yaml:"filters"`
+}
+
+type DisputeGameVerifier struct {
+	factory    common.Address
+	factoryABI abi.ABI
```

golangci-lint (1.62.2)

field `factoryABI` is unused

(unused)